Browser `tracingOrigins` is deprecated and replaced with `tracePropagationTargets` (#5759)
getsentry/sentry-docs: src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx

Supported options:

> ### tracePropagationTargets

Reviewer: Can we add a note saying that this was previously called `tracingOrigins`?

Supported options:

### tracePropagationTargets

> Previously called `tracingOrigins`.

Reviewer: ```suggestion
<Note>

The `tracingOrigins` option was was renamed to `tracePropagationTargets` and deprecated in [version `7.19.0`](https://github.com/getsentry/sentry-javascript/releases/tag/7.19.0) of the JavaScript SDK. `tracingOrigins` will be removed in the v8 version.

</Note>
```

Supported options:

### tracePropagationTargets

<Note>

> The `tracingOrigins` option was was renamed to `tracePropagationTargets` and deprecated in [version `7.19.0`](https://github.com/getsentry/sentry-javascript/releases/tag/7.19.0) of the JavaScript SDK. `tracingOrigins` will be removed in the v8 version.

Reviewer 2: ```suggestion
The `tracingOrigins` option was renamed `tracePropagationTargets` and deprecated in [version `7.19.0`](https://github.com/getsentry/sentry-javascript/releases/tag/7.19.0) of the JavaScript SDK. `tracingOrigins` will be removed in version 8.
```

</Note>

A list of strings and regular expressions. The JavaScript SDK will attach the `sentry-trace` and `baggage` headers to all outgoing XHR/fetch requests whose destination contains a string in the list or matches a regex in the list. If your frontend is making requests to a different domain, you'll need to add it there to propagate the `sentry-trace` and `baggage` headers to the backend services, which is required to link transactions together as part of a single trace.

> **The `tracePropagationTargets` option matches against the whole request URL, not just the domain. Using stricter regex to match certain parts of the URL ensures that requests do not unnecessarily have the additional headers attached.**

Reviewer 2: ```suggestion
**The `tracePropagationTargets` option matches the entire request URL, not just the domain. Using stricter regex to match certain parts of the URL ensures that requests don't unnecessarily have additional headers attached.**
```

### shouldCreateSpanForRequest

> This function can be used to filter out unwanted spans such as XHR's running
> health checks or something similar. If this function is not specified, spans
> will be created for all requests.

Reviewer 2: ```suggestion
This function can be used to filter out unwanted spans such as XHRs running
health checks or something similar. If this function isn't specified, spans
will be created for all requests.
```
